website_form_recaptcha/tests/test_captcha.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+# © 2015-TODAY LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp.tests.common import TransactionCase
+from openerp.exceptions import ValidationError
+import mock
+
+imp_model = \
+    'openerp.addons.website_form_recaptcha.models.website_form_recaptcha'
+imp_requests = '%s.requests' % imp_model
+
+
+class TestCaptcha(TransactionCase):
+
+    def setUp(self):
+        super(TestCaptcha, self).setUp()
+        self.model_obj = self.env['website.form.recaptcha']
+        self.secret_key = self.env.ref(
+            'website_form_recaptcha.recaptcha_key_secret'
+        ).value
+        self.validate_vars = 'T1', 'T2'
+
+    @mock.patch(imp_requests)
```

Comment:
Interesting. I still don't understand the code but nevermind, I'll look into it when I need something similar. Thanks!